docs/resources/artifact_property_webhook.md

```diff
@@ -46,14 +46,14 @@ The following arguments are supported:
 
 * `key` - (Required) The identity key of the webhook. Must be between 2 and 200 characters. Cannot contain spaces.
 * `description` - (Optional) Webhook description. Max length 1000 characters.
-* `enabled` - (Optional) Status of webhook. Default to 'true'.
-* `event_types` - (Required) List of Events in Artifactory, Distribution, Release Bundle that function as the event trigger for the Webhook. Allow values: "added", "deleted".
+* `enabled` - (Optional) Status of webhook. Default to `true`.
+* `event_types` - (Required) List of Events in Artifactory, Distribution, Release Bundle that function as the event trigger for the Webhook. Allow values: `added`, `deleted`.
 * `criteria` - (Required) Specifies where the webhook will be applied on which repositories.
   * `any_local` - (Required) Trigger on any local repo.
   * `any_remote` - (Required) Trigger on any remote repo.
   * `repo_keys` - (Required) Trigger on this list of repo keys.
-  * `include_patterns` - (Optional) Simple comma separated wildcard patterns for repository artifact paths (with no leading slash).\n Ant-style path expressions are supported (*, *\*, ?).\nFor example: "org/apache/**".
-  * `exclude_patterns` - (Optional) Simple comma separated wildcard patterns for repository artifact paths (with no leading slash).\n Ant-style path expressions are supported (*, *\*, ?).\nFor example: "org/apache/**".
+  * `include_patterns` - (Optional) Simple comma separated wildcard patterns for repository artifact paths (with no leading slash).\n Ant-style path expressions are supported (*, *\*, ?).\nFor example: `org/apache/**`.
```

Comment:
/n is rendered as is in the Terraform documentation. 
<img width="643" alt="image" src="https://user-images.githubusercontent.com/55418063/223570711-4b086d00-f18f-4505-8078-67758774e500.png">